### PR TITLE
Integrate analysis into workflow and dump best model for testing

### PR DESCRIPTION
**Pull request by cmmclaug**

### CODE REVIEW

**Review by aidanmattrick — APPROVED** [review]
Looks good to me Craig!

File: Makefile [hunk 1]
```diff
@@ -28,31 +28,43 @@ results/tables/feature_overview.pkl : \
...
-	rm -f data/processed/clean_characters_test.csv
-	rm -f data/processed/clean_characters_deploy.csv
-	rm -f results/figures/alignment_over_time.png
-	rm -f results/figures/alignment_vs_features.png
-	rm -f results/figures/appearances_by_alignment.png
-	rm -f results/tables/dataset_overview.pkl
-	rm -f results/tables/feature_overview.pkl
-	rm -f report/eda_profile_report.html
\ No newline at end of file
+	rm -f data/raw/*
```
Comment (zmerpez) [inline Makefile:63]: That is a nice and more clean way. Good call.

**Review by zmerpez — APPROVED** [review]
Thanks, those scripts are more readable and user friendly.

**Review by ssyayayy — APPROVED** [review]
Thanks Craig!

**Pull request merged**